react/SelectBox/Readme.md

```diff
@@ -25,6 +25,53 @@ const options = [
 <SelectBox options={options} menuIsOpen />
 ```
 
+### Props
+
+#### `fullwidth`
+
+Set the select to spread to 100% of the available width (default: `false`).
+
+```
+const options = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry', isDisabled: true },
+  { value: 'vanilla', label: 'Vanilla' },
+  { value: 'long', label: 'Salt and vinegar crisps with vegamite and brussel sprouts, double chai latte sauce' },
+];
+
+<SelectBox options={options} fullwidth />
+```
```

Comment:
Since it deals with `size`, could we simplify the API by having `size='full'` instead of `fullwidth ?

Reply:
The idea here is to use the same API as Field component. In Field, size is used for component's height.

Reply:
OK thanks.